src/extension/wps/src/main/java/au/org/emii/wps/gogoduck/parameter/SubsetParameters.java

```diff
@@ -73,6 +64,7 @@ public static SubsetParameters parse(String subset) {
         ParameterRange timeRange = subsets.get("TIME");
         CalendarDateRange calendarDateRange = null;
 
+        //Validation
```

Comment:
Perhaps include expected date/time format in exception so requestor has some idea what they need to provide.

Could also include this in the process documentation included in https://github.com/aodn/geoserver-build/blob/master/src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java#L55 (appears in WPS DescribeProcess request)